src/controllers/reviewController.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from "express";
+import Review from "../models/Review";
+
+export const createReview = async (req: Request, res: Response) => {
+    const { content, postId, parentReview } = req.body;
+    const userId = req.user.userId;
+
+    if (!userId) {
+        return res.status(401).send({ error: 'Unauthorized' });
+    }
```

Comment:
This logic is good. Now you have to push this review into the post model using the post id.